YML.md

```diff
@@ -1,12 +1,21 @@
 # About `.yml` files
 
-The `recipes/meta` directory in this repository contains `.yml` files.
+The `recipes/meta` directory in this repository contains `.yml` files. These are used by the `Recipe` to simplify the creation of AppImages, especially by using pre-built binaries from Debian packages, both to save time for creating and building an AppImage.
```

Comment:
Possibly we should rename "Recipe". I think the term made sense when it was a single bash script that did the conversion. With the "meta Recipe using yml files" - not so much anymore. What would be a better name? package2AppImage?